components/script/script_module.rs

```diff
@@ -1277,7 +1277,7 @@ impl FetchResponseListener for ModuleContext {
 
     fn process_csp_violations(&mut self, _request_id: RequestId, violations: Vec<csp::Violation>) {
         let global = &self.resource_timing_global();
-        global.report_csp_violations(violations);
+        global.report_csp_violations(violations, None);
```

Comment:
If we have a `<script type="module">` that fails a CSP check, I think we would want to provide the element here. We should file a followup issue about that.

Reply:
I retract this suggestion. Non-inline module scripts are similar to image elements.